pkg/handler/handler.go

```diff
@@ -205,6 +207,31 @@ func (handler *Handler) Update(obj client.Object) error {
 	return nil
 }
 
+func (handler *Handler) GetBootstrapUser(instance client.Object) (*corev1.Secret, error) {
```

Comment:
I think this func should in `controllers/apps/v1beta4`, not in here

Reply:
done